src/romaine/parser/__init__.py

```diff
@@ -0,0 +1,34 @@
+from romaine.parser import (
+    simple,
+    multiline,
+    step,
+    section,
+    feature,
+    )
+from romaine.parser.exceptions import (  # noqa
+    UnclosedPythonishString,
+    MalformedTableError,
+    )
+
+
+class Parser(object):
+    """
+        Gherkin parser for romaine core.
```

Comment:
I think we both have the same reaction to each other's preferred convention ;)

Obviously I'd prefer it my way, but I can do that as a separate PR once all of this is in.